src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -356,9 +356,10 @@ public function getGlobalScopes()
 	 * Register an observer with the Model.
 	 *
 	 * @param  object|string  $class
+	 * @param  int            $priority
```

Comment:
Please don't align this. Just use exactly 2 spaces either side of int. :)

Please squash to one commit after.